quarkus/service/build.gradle.kts

```diff
@@ -87,24 +87,19 @@ dependencies {
 
   implementation(libs.jakarta.servlet.api)
 
-  testFixturesApi(project(":polaris-tests"))
+  testFixturesApi(project(":polaris-tests")) {
+    // exclude all spark dependencies
+    exclude(group = "org.apache.iceberg", module = "iceberg-spark-3.5_2.12")
```

Comment:
Would it make sense to split `polaris-tests` into a general part (that only uses Iceberg Core) and a Spark-specific part (instead of excluding spark jars)?